python/hail/expr/functions.py

```diff
@@ -1548,3 +1548,48 @@ def to_str(x):
     :class:`.StringExpression`
     """
     return _func("str", TString(), x)
+
```

Comment:
should be expr_list. Check in the body:

```
if not is_numeric(pl.dtype.element_type):
    raise TypeError("expected an expression of type 'Array[numeric]', found '{}'".format(pl.dtype))
```

Reply:
These routines require Array[Int], won't work as written with numeric.  That might be useful, but can we leave it for another time?

Reply:
Yeah, we can. We should have a way to do this correctly within typecheck, so it's fine to wait on the correct thing.